tests/fixtures/headless.js

```diff
@@ -0,0 +1,174 @@
+
+var read = require('fs').readFileSync;
+var _ = require('lodash');
+
+function headless (benv, binding) {
+  var self = binding;
+  function root ( ) {
+    return benv;
+  }
+
+  function init (opts, callback) {
+    var localStorage = opts.localStorage || './localstorage';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['localStorage' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)